app.js

```diff
@@ -0,0 +1,13 @@
+require('dotenv').config()
+const express = require('express')
+const app = express()
+const port = 3000
+const router = require ("./routes")
+
+app.use (express.urlencoded({extended:false}))
+app.use ('/', router)
+app.use(express.json())
```

Comment:
ini kan body parser, jadi dia tujuannya sama dengan urlencoded
```suggestion
app.use (express.urlencoded({extended:false}))
app.use(express.json())
app.use ('/', router)
```

Reply:
siap ka . wkkwkw ini mah kayanya masalah di ilmu fase 1 saya. maapin ka